ctools/defragment_sharded_collection.py

```diff
@@ -83,7 +83,7 @@ async def split_chunk(self, chunk, maxChunkSize_kb):
 
         if len(res['splitKeys']) > 0:
             await self.cluster.adminDb.command({
-                    'splitChunk': self.name,
+                    'split': self.name,
                     'bounds': [chunk['min'], chunk['max']],
                     'splitKeys': res['splitKeys']
```

Comment:
From our official documentation it seems that `split` command doesn't support the `splitKeys` parameter

Reply:
using the shard connection now